how-to/customize-workspace/client/src/framework/platform/platform.ts

```diff
@@ -60,6 +60,19 @@ async function setupPlatform(_?: PlatformProviderOptions): Promise<boolean> {
 	conditionsProvider.registerCondition("sharing", async () => isShareEnabled(), false);
 	conditionsProvider.registerCondition("themed", async () => supportsColorSchemes(), false);
 
+	const aboutVersionWindow = await versionProvider.getAboutWindow();
```

Comment:
Maybe move this code to a method in version.ts, platform shouldnt need to know about version windows etc

Reply:
It was originally in the versionProvider but then I didn't want the version provider to make the decision. Moved it to platform so the platform can make the call based on the data at hand but will be moving this into a module.